src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator/CalculateHelper.cs

```diff
@@ -18,6 +20,7 @@ public static class CalculateHelper
             @"sinh\s*\(|cosh\s*\(|tanh\s*\(|arsinh\s*\(|arcosh\s*\(|artanh\s*\(|" +
             @"pi|" +
             @"==|~=|&&|\|\||" +
+            @"((-?(\d+(\.\d*)?)|-?(\.\d+))[E](-?\d+\.*\d*))|" + /* expression from CheckScientificNotation between parenthesis */
```

Comment:
should it be updated here as well? only integer after E

Reply:
Whoops. Fixed!

Reply:
thanks :) Change looks good to me, I'll just give it a test and then it'll be good to go! :)